cookbooks/simple_ice_box.prm

```diff
@@ -0,0 +1,174 @@
+# A simple setup for using the boundary heat flux in a ice viscoplatic model.
```

Comment:
I also find it helpful to add a brief statement regarding what previous studies (if any) the cookbook is based on and a broad overview of the relevant application (e.g., icy planet, etc).